storage_service/locations/models/package.py

```diff
@@ -63,6 +63,7 @@ class Package(models.Model):
         (DEPOSIT, 'FEDORA Deposit')
     )
     package_type = models.CharField(max_length=8, choices=PACKAGE_TYPE_CHOICES)
+    related_packages = models.ManyToManyField('self')
```

Comment:
This line should probably be in the other commit. Look at git rebase, or ask if you aren't sure how to move it.